games-fps/prboom-plus/prboom_plus-2.5.1.4.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="Enhanced clone of the classic first-person shooter Doom"
+DESCRIPTION="PrBoom+ is a Doom source port developed from the original PrBoom
+ project by Andrey Budko"
+HOMEPAGE="http://prboom-plus.sourceforge.net"
+COPYRIGHT="2016 Andrey Budko et al."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/prboom-plus/prboom-plus-$portVersion.tar.gz"
+CHECKSUM_SHA256="e0e2241d174839b107f1a42b191aa9895dc6749e477cbc850ad520fba2353b94"
+SOURCE_DIR="prboom-plus-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+
+commandBinDir="$binDir"
+if [ "$targetArchitecture" = "x86_gcc2" ]; then
+	SECONDARY_ARCHITECTURES="x86"
+	commandBinDir="$prefix/bin"
+fi
+
+PROVIDES="
+	cmd:prboom_plus = $portVersion
+	cmd:prboom_plus_game_server = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libSDL_1.2$secondaryArchSuffix
+	lib:libSDL_net_1.2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libSDL_1.2$secondaryArchSuffix
+	devel:libSDL_net_1.2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -fi
```

Comment:
Is `autoreconf` necessary? The source archive already contain a `configure` script.

Reply:
@korli reccomended it since autoconf is automatically triggered

Reply:
it shouldn't be required for recent projects, for which no patch of configure.ac is needed.